packages/terra-clinical-result/src/_BloodPressureDisplay.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
we had a discussion in the past about removing underscored components and explicitly creating a private directory to contain private functionality. i can't find the discussion, but we've followed through in terra-application https://github.com/cerner/terra-application/tree/master/src/application-base/private

it doesn't have to be in this pull request, but we should plan on following that pattern in the future